site/smart-accounts/modular-account/multisig-plugin/usage-guide.md

```diff
@@ -0,0 +1,153 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Modular Account with Multisig Plugin • Getting started
+  - - meta
+    - name: description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+  - - meta
+    - property: og:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+  - - meta
+    - name: twitter:title
+      content: Modular Account with Multisig Plugin • Getting started
+  - - meta
+    - name: twitter:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+---
+
+# Getting started with Modular Account with Multisig Plugin
+
+To start, follow the guide to set up Modular Account [here](../getting-started.md).
+
+## Modular Account with Multisig Plugin Actions
+
+### Creating a Account Client
+
+The first step is to create a Multisig Modular Account client 
+
+```ts
+import { LocalAccountSigner } from "@alchemy/aa-core";
+import { createMultisigAccountAlchemyClient } from "@alchemy/aa-alchemy";
+
+// Creating a 3/3 multisig account 
+const signers = [LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 0 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 1 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 2 }
+  )];
+
+const threshold = 3n;
+
+const owners = await Promise.all(
+  signers.map(async (s) => s.getAddress()),
```

Comment:
(Nit) There's no need to mark a function `async` if you never use `await` within it's body and it's already returning a promise. Can change to
```ts
signers.map((s) -> s.getAddress())
```